docs/application/tizen-studio/extension-sdk/in-app-purchase.md

```diff
@@ -271,454 +280,469 @@ if (app_control != NULL)    app_control_destroy(app_control);
+  | `_transactionId` | Transaction ID            | Transaction ID, such as 1 or 2.The ID is used to track a transaction between requests.This information is mandatory. |
+  | `_itemId`        | Item ID                   | Item ID, such as 000000003501.An item ID is associated with a specific item in Tizen Store Seller Office.You can retrieve a list of items available for purchase using the `org.tizen.inapppurchase.iapservice` application ID.This information is mandatory. |
+  | `_itemGroupId`   | Group ID                  | Group ID, such as 100000001455.A group ID is associated with a specific collection of items in Tizen Store Seller Office. You need to register your group ID in Tizen Store Seller Office first.This information is mandatory. |
+  | `_languageCd`    | Language code             | Language code, such as `eng` or `rus`.The language code conforms to ISO 639-2, which uses 3-character codes.The language code is associated with the display language of the item details in Tizen Store Seller Office.This information is optional.<br> **Note**<br> `_itemName` is higher priority than `_languageCd`. |
+  | `_itemName`      | Item name                 | Item name, such as "Sword" or "칼".Specify your item name with this key, or leave the key empty to use the default name on the seller site's purchase page.This information is optional. |
+  | `_mcc`           | Mobile country code (MCC) | Mobile country code, such as 250.MMCs can only be used in developer mode. You can retrieve a list of available MCCs using the get country list operation.This information is optional.This parameter has been deprecated. |
+  | `_mnc`           | Mobile network code (MNC) | Mobile network code, such as 01.MNCs can only be used in developer mode.This information is optional.This parameter has been deprecated. |
 
 - **Example code for Purchase**
- ```
- app_control_h app_control;
- int rt = app_control_create(&app_control);
- if (rt == APP_CONTROL_ERROR_NONE) {    
-      app_control_set_app_id(app_control, " org.tizen.inapppurchase.iapclient");    
-      app_control_set_operation(app_control, "http://tizen.org/appcontrol/operation/iapv2/purchase");    
-      app_control_add_extra_data(app_control, "_mode", "0");  
-      app_control_add_extra_data(app_control, "_itemId", "000000000001");    
-      app_control_add_extra_data(app_control, "_itemGroupId", "100000000012");    
-      app_control_add_extra_data(app_control, "_languageCd", "ENG"); /* Optional */    
-      app_control_add_extra_data(app_control, "_itemName", "Item 1"); /* Optional */   
-      rt = app_control_send_launch_request(app_control, get_purchase_cb, NULL);
- }
-
- if (app_control != NULL)    
-      app_control_destroy(app_control);
- ```
+       ```
```

Comment:
The code block style is not applied well.

Reply:
done